### PR TITLE
feat: added navbar

### PR DESCRIPTION
**Pull request by KostLinux**

### CODE REVIEW

File: validation_report.log
Comment (Matrix278) [inline validation_report.log:1]: Ignore the file?

**Comment by Matrix278** [comment]
@KostLinux @Dan-I-El Checked locally, that's what I found that can be done better:
1. The U-Switch logo on navbar as a button that does unnecessary reload page, but I guess could be just the same as home navbar link (#cctv) - http://127.0.0.1:5500/#cctv

2. On Russian the link to AJAX is only within the AJAX word, but on Estonian it's within 2 words AJAX tooteid. Should be the same for both languages, as I think.
image
image

3. On the navbar there is no active link coloring, like if I am on Teenused, I don't see other color on it, could be done also. Or maybe hover color at least if even needed.
image

4. The footer of copyright should have configured 2024 to be updated next year -  © U SWITCH OÜ 2024 -> © U SWITCH OÜ 2024-2025
I have it done on my portfolio, you can check it out with JS script - https://github.com/Matrix278/portfolio/blob/master/index.html
https://github.com/Matrix278/portfolio/blob/master/js/main.js#L27C1-L29C1

5. Two images on the back, is it ok? Shouldn't it be just one image?
image

6. Should the facebook and instagram making new tab with target="_blank" and not just redirect? Like it's done for the ajax page.

7. And was it ok for them to have the navbar like from the right start of the screen and to the left end of the screen? Not within the center how I remember they had it? That's about the container. 
image

File: assets/css/main.css [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    font-family: "Libre Baskerville", serif;
+    font-size: 24px;
+    font-style: normal;
+    font-weight: 400;
+    height: auto;
+    letter-spacing: 0.6px;
+    line-height: 31.2px;
+    opacity: 1;
+    overflow-wrap: break-word;
+    text-align: center;
```
Comment (Matrix278) [inline assets/css/main.css]: Duplicate of 'text-align: center;', delete please.

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+#mobile-menu.show a:nth-child(1) {
+    transition-delay: 0.1s;
+}
+
+#mobile-menu.show a:nth-child(2) {
+    transition-delay: 0.2s;
+}
+
+
```
Comment (Matrix278) [inline assets/css/main.css]: Format the file, to remove unnecessary 2 newlines and fix tabs

File: assets/js/main.js [hunk 3]
```diff
@@ -0,0 +1,164 @@
+document.addEventListener('DOMContentLoaded', function() {
+    // Mobile menu toggle
+    document.getElementById('menu-button').addEventListener('click', function() {
+        var menu = document.getElementById('mobile-menu');
```
Comment (Matrix278) [inline assets/js/main.js]:
```suggestion
        let menu = document.getElementById('mobile-menu');
```

File: assets/js/main.js [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+            setTimeout(function() {
+                menu.classList.add('hidden');
+            }, 300); // Delay to match the transition duration
+        }
+    });
+
+    // Close mobile menu when a menu item is clicked
+    document.querySelectorAll('.mobile-menu-item').forEach(function(item) {
+        item.addEventListener('click', function() {
+            var menu = document.getElementById('mobile-menu');
```
Comment (Matrix278) [inline assets/js/main.js]:
```suggestion
            let menu = document.getElementById('mobile-menu');
```

File: assets/js/main.js [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+                });
+            })
+            .catch((error) => {
+                console.error("Error loading translations:", error);
+            });
+    }
+
+    // Load translations when the page loads
+    loadTranslations("ee"); // Default language
+});
```
Comment (Matrix278) [inline assets/js/main.js]: Format the js code also

File: translations/ee.json [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    "about_title": "Meist",
+    "about_paragraph_1": "Ettevõte U Switch on spetsialiseerunud nutimajade ehitamisele, nutimoodulite paigaldamisele ning kvaliteetsete seadmete müügile. Oleme Ajax Security System kööstööpartnerid, tänu millele saame pidevalt nende poolt täiendavaid koolitusi, tagades sellega oma kliendile kõige uuema ja turvalisema lahenduse.",
+    "about_paragraph_2": "Meie eesmärk on muuta Teie kodu või äripind turvalisemaks ning lihtsustada igapäevaseid toiminguid nutikate seadmete abil. Oleme pühendunud pakkuma parimat teenust ning kvaliteetseid tooteid. Liituge meiega ja laske meil teha Teie elu lihtsamaks ja turvalisemaks!",
+    "about_paragraph_3": "                      - Denis Kitšigin ja Pavel Borovik (Juhatuse liikmed)",
+
+    "contact_title": "Kontakt",
+    "contact_register_number": "Registrikood: 16795378",
+
+    "book_time_button":"Broneeri konsultatsioon"
+}
```
Comment (Matrix278) [inline translations/ee.json:23]: I checked that the JSONs also could be formated better:
image

File: translations/ru.json [hunk 7]
```diff
@@ -0,0 +1,31 @@
+{
+    "navbar_home": "Главная страница",
+    "navbar_services": "Услуги",
+    "navbar_about": "О нас",
+    "navbar_contact": "Контакт",
+
+    "cctv_title": "Создание безопасных решений для умного дома - Это наше искусство",
+    "cctv_subtitle": "Выбирайте надёжные и умные решения от U SWITCH , чтобы сделать свой дом комфортным и безопасным.",
+
+    "services_header": "Мы предлагаем продукцию <a class='underline' href='https://ajax.systems/' target='_blank' title='https://ajax.systems/'>AJAX</a> с использованием современных устройств безопастности и умных модулей",
```
Comment (Matrix278) [inline translations/ru.json]:
```suggestion
    "services_header": "Мы предлагаем продукцию <a class='underline' href='https://ajax.systems/' target='_blank' title='https://ajax.systems/'>AJAX</a> с использованием современных устройств безопасности и умных модулей",
```

File: translations/ee.json [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    "services_smarthouse_solutions": "Nutimaja lahendused",
+    "services_smarthouse_solutions_description": "Teostame töid alati lähtudes kliendi soovidest ja tagasisidest",
+
+    "devices_sell": "Seadmete müük",
+    "devices_sell_description": "Pakume tipptasemel juhtmevabu tooteid, mille on innovaatiline disain",
+
+    "about_title": "Meist",
+    "about_paragraph_1": "Ettevõte U Switch on spetsialiseerunud nutimajade ehitamisele, nutimoodulite paigaldamisele ning kvaliteetsete seadmete müügile. Oleme Ajax Security System kööstööpartnerid, tänu millele saame pidevalt nende poolt täiendavaid koolitusi, tagades sellega oma kliendile kõige uuema ja turvalisema lahenduse.",
+    "about_paragraph_2": "Meie eesmärk on muuta Teie kodu või äripind turvalisemaks ning lihtsustada igapäevaseid toiminguid nutikate seadmete abil. Oleme pühendunud pakkuma parimat teenust ning kvaliteetseid tooteid. Liituge meiega ja laske meil teha Teie elu lihtsamaks ja turvalisemaks!",
+    "about_paragraph_3": "                      - Denis Kitšigin ja Pavel Borovik (Juhatuse liikmed)",
```
Comment (Matrix278) [inline translations/ee.json]: What is happening with the big space within `about_paragraph_3` translation key? Although in russian other way.
    "about_paragraph_3": "                      - Denis Kitšigin ja Pavel Borovik (Juhatuse liikmed)",

**Review by Matrix278 — CHANGES_REQUESTED** [review]
Added comments

**Comment by KostLinux** [comment]
@Matrix278 for them it was okay, that navbar will be from left to the right. But for the content (except the first cctv section) you can add the container to contribute.
Additionally could i ask on which view you see doubled pictures? `Two images on the back, is it ok? Shouldn't it be just one image?` 
Cause whenever i try to reproduce, i cannot find anything ;(

Other stuff are done.

**Review by Matrix278 — APPROVED** [review]

**Comment by Dan-I-El** [comment]
I don't even no how it should look so LGTM

**Review by Dan-I-El — APPROVED** [review]

**Pull request merged**